Selenium.Axe/Selenium.Axe.Test/AxeBuilderTest.cs

```diff
@@ -11,6 +14,36 @@ namespace Selenium.Axe.Test
     [TestClass]
     public class AxeBuilderTest
     {
+        private Mock<IWebDriver> webDriverMock;
+        private Mock<IJavaScriptExecutor> jsExecutorMock;
+        private Mock<ITargetLocator> targetLocatorMock;
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
```

Comment:
We reuse this in a few places; should we make it part of the impl assembly (`DefaultJsonSerializerSettings` or something) to make sure it stays in sync?